src/python/grpcio_tests/tests_aio/unit/_test_server.py

```diff
@@ -18,38 +18,27 @@
 from time import sleep
 
 import grpc
+from grpc.experimental import aio
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2_grpc
 from tests.unit.framework.common import test_constants
 
 
-# TODO (https://github.com/grpc/grpc/issues/19762)
-# Change for an asynchronous server version once it's implemented.
 class TestServiceServicer(test_pb2_grpc.TestServiceServicer):
```

Comment:
Now that this is used as a library and not a binary, this class should probably become private, i.e. `_TestServiceServicer`.

Reply:
Good catch!